Yoast/wordpress-seo
# Fixed rel next and rel prev for static home pages

## Summary


This PR can be summarized in the following changelog entry:

* N\A

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Create a new page with at least 3 pages pages (You can use the page break block for that).
* Under `Settings` > `Reading` > `Your homepage displays`, set it to `A static page` and choose the page you just created.
* If https://github.com/Yoast/wordpress-seo/issues/13841 hasn't been fixed: Edit the page you selected, make a change and update it.
* Go to your homepage.
  * Make sure it has a `link rel="next"` meta tag pointing to the next page, make sure the link works.
  * Make sure it has no `link rel="prev"` meta tag.
* Go to page 2 for the page.
  * Make sure it has a `link rel="prev"` meta tag pointing to the root URL, make sure the link works.
  * Make sure it has a `link rel="next"` meta tag pointing to the next page, make sure the link works.
* Go to the last page
  * Make sure it has a `link rel="prev"` meta tag pointing to the previous page, make sure the link works.
  * Make sure it has no `link rel="next"` meta tag.

## UI changes

## Documentation

## Quality assurance


Fixes
